services/apps/alcs/src/alcs/application/application-submission-status/application-submission-status.service.ts

```diff
@@ -152,10 +152,12 @@ export class ApplicationSubmissionStatusService {
           SUBMISSION_STATUS.IN_REVIEW_BY_ALC,
         ]),
         emailSentDate: IsNull(),
-        effectiveDate: LessThanOrEqual(date), // this will get only statuses <= today since the status service converts all days to .startOf('day')
+        effectiveDate: And(Not(IsNull()), LessThan(date)), // this will get only statuses < today+1 since the status service converts all days to .startOf('day')
```

Comment:
Tomorrow might be a clearer way than saying today +1, but why do we want to include statuses from tomorrow?

Reply:
They will not be included since it checks effectiveDate < tomorrow start of day.